dataactvalidator/config/sqlrules/a10_appropriations.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+    approp.row_number,
+    approp.borrowing_authority_amount_cpe,
+    SUM(sf.amount) as total_amount
+FROM appropriation as approp
+    INNER JOIN sf_133 as sf ON approp.tas = sf.tas
+    INNER JOIN submission as sub ON approp.submission_id = sub.submission_id AND
+        sf.period = sub.reporting_fiscal_period AND
+        sf.fiscal_year = sub.reporting_fiscal_year
+WHERE approp.submission_id = {} AND
+    sf.line in (1340, 1440)
+GROUP BY approp.row_number, approp.borrowing_authority_amount_cpe
```

Comment:
when using GROUP BY, the column that need to be grouped on are all columns NOT in the aggregate function. So in this case, we're only summing on `sf.amount` so all other columns must be included in the GROUP BY statement